kaiju/Avatar.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Avatar",
+  "library": "terra-avatar",
+  "display": "Avatar",
+  "description": "A circular frame to diplay an image or text",
+  "group": "Atoms::Other",
+  "documentation": "http://engineering.cerner.com/terra-core/#/site/avatar",
```

Comment:
This is referencing the terra-core engineering site. This will be updated when terra-ui is updated to include avatar

Reply:
It's deployed now.
http://engineering.cerner.com/terra-ui/#/site/components/core/site/avatar/index